### DISCUSSION TITLE
How to update multiple fields receiving on action function an stringify object [REMIX]

### DISCUSSION THREAD

**Discussion by Joseamica** [prisma/prisma#17487] [Q&A]
On every delivery food app, when someone chooses the dish item, this contains multiple modifiers groups and modifiers, for example:
You select Hotcakes,  modal opens with all the dish information, including these modifiers. What type of milk do you want? => "Almond" "Light" "Normal". and there is another modifier Size => "Big" "Small".

In this example, dishes can share multiple modifiers, but some modifiers may be unique.
I'm struggling to do this dynamically because the restaurant would want to give different modifiers for each dish, and there are a lot of restaurants.

Right now when the user chooses, it returns to my action function an object stringify, but I can't manage to connect these multiple ids to multiple fields chosen by the user to connect to the DB.

![Screen_Shot_2023-01-23_at_10 31 45](https://user-images.githubusercontent.com/15864799/214117642-9834d42b-845e-4002-a5a0-09cb62ac0ecc.png)

**Comment by nurul3101** [comment]
Hello @Joseamica 👋 

I believe this is related to this previous GitHub Discussion that you opened:
- https://github.com/prisma/prisma/discussions/17259

Continuing the conversation from the previous discussion, you are not looking for [IN](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#in) operator, right? Also, sharing your prisma query would be helpful.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
